homeassistant/components/modbus/modbus.py

```diff
@@ -136,7 +136,7 @@ async def async_modbus_setup(
         hubs = hass.data[DOMAIN]
         for name in hubs:
             if not await hubs[name].async_setup():
-                return False
+                return False  # pragma: no cover
```

Comment:
I don't think we accept `no cover`
You should add a test instead, or leave uncovered.

Reply:
I am working on adding a test...but what is the difference between using "no cover" and publicly telling it is not covered or do it secretly by ignoring it...seems strange to me.

Reply:
Done.